app/controllers/admin/events_controller.rb

```diff
@@ -15,5 +15,11 @@ class EventsController < Admin::ApplicationController
 
     # See https://administrate-docs.herokuapp.com/customizing_controller_actions
     # for more information
+
+    private
+
+    def resource_params
```

Comment:
That method is duplicated in three controllers. I think it would be a good idea to create a concern (or a good ol' ruby module) to include. If we want to modify it (if we want to let the admin override the value to write on behalf of another user for example), we'll do it once.